vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2444,21 +2444,27 @@ def _reshape_kv_cache_tensors(
                 elif isinstance(kv_cache_spec, MambaSpec):
                     raw_tensor = kv_cache_raw_tensors[layer_name]
                     dtype = kv_cache_spec.dtype
+                    page_size = kv_cache_spec.page_size_bytes // get_dtype_size(
```

Comment:
```suggestion
                    num_element_per_page = kv_cache_spec.page_size_bytes // get_dtype_size(
```
This variable name is more clear for me

Reply:
Agree, will change